synapse/app/homeserver.py

```diff
@@ -300,6 +302,14 @@ def run_startup_checks(self, db_conn, database_engine):
             quit_with_error(e.message)
 
 
+# Gauges to expose monthly active user control metrics
+current_mau_gauge = Gauge("synapse_admin_current_mau", "Current MAU")
+max_mau_value_gauge = Gauge("synapse_admin_max_mau_value", "MAU Limit")
+limit_usage_by_mau_gauge = Gauge(
+    "synapse_admin_limit_usage_by_mau", "MAU Limiting enabled"
+)
```

Comment:
I would probably not bother having `limit_usage_by_mau_gauge`, and simply set it to 0 or null if limit isn't enabled